.pre-commit-config.yaml

```diff
@@ -10,7 +10,7 @@
     -   id:  double-quote-string-fixer
     -   id:  end-of-file-fixer
     -   id:  flake8
-        exclude: '^blog/migrations/|^mysite/settings.py|^mysite/wsgi.py'
+        exclude: '^blog/migrations/'
```

Comment:
Quotes are usually not necessary in YAML, unless you wrap some arbitrary symbols.